src/test/java/seedu/address/model/country/CountryManagerTest.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.country;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Locale;
+
+import org.junit.jupiter.api.Test;
+
+public class CountryManagerTest {
+
+    //TODO: Add more tests if decide to include checking for 3-letter Country Code
+    private static final String[] countryCodes = Locale.getISOCountries();
```

Comment:
Constant names must be all uppercase using underscore to separate words.

Reply:
Yup good catch